README.md

```diff
@@ -44,14 +44,15 @@ We largely follow the ROS coding guidelines, with a few noteable exceptions. To
 To install the linters:
 
 ```
-$ sudo apt install clang-format-7 clang-tidy-7 yapf
+$ sudo apt install clang-format-7 clang-tidy-7
+$ pip install yapf pylint
 ```
 
-To run the linters:
+To run the formatters:
 
 ```
 $ find . -name "*.h" -o -name "*.cpp" | xargs clang-format-7 -i -style=file
-$ yapf -ir .
+$ python -m yapf -ir .
```

Comment:
This addition is unnecessary

Reply:
Don't think it is - I changed the install to use pip instead of apt, since apt is update a lot less frequently and the most recent version in apt is 6 versions older than in pip. And when we install with pip, it doesn't get added to the path.

Reply:
It does get added to the path, I'm not sure where you are getting this from.

Reply:
My computer :P

Maybe there's something fried about my setup

Reply:
Python packages should be in your path, at least, it is on macOS by default. I don't have a linux machine to test with right now but I have one I can use tomorrow where I can use a fresh install of python and see if it's added by default.

Reply:
For history: Recent versions of Bash (and Debian and Ubuntu) have issues where `~/.local/bin` isn't on the path. Workarounds are to either add it to the path (prefered) or run the program with `python -m`.